src/aws/flb_aws_util.c

```diff
@@ -309,6 +310,14 @@ struct flb_http_client *request_do(struct flb_aws_client *aws_client,
         goto error;
     }
 
+    // Set max response buffer size for the http client. 
+    // Try setting resp.max_size to 0 to allow buffer to be resized. 0 means unlimited.
+    if (!flb_http_buffer_size(c, MAX_RESP_BUFFER_SIZE)) {
+        flb_debug("[aws_http_client] max response buffer size was increased to %zu", MAX_RESP_BUFFER_SIZE);
```

Comment:
as discussed in slack, this will lead to a very confusing message for the customer since we set `0` to mean unlimited. please make the message very clear.

Reply:
Since this is not a configurable value and the log message is repeated on every chunk of data fed to the output plugin (correct me if I am wrong), we should make this a trace message instead of debug.

Reply:
+1 a user doesn't need to see this if it succeeds

Reply:
I think we can only print a trace if it fails. We do have a need to print a trace when it succeds? Or, we can just follow other plugin's styly-- print nothing. Because user can see this limit in other places. Thoughts?

Reply:
I agree that for success we should print nothing.

let's print a warning if it fails? since we want users to see that and may be report it to us in an issue, since failure should not happen?

Reply:
We need an error if this fails. This is true for AWS plugins.